webgpu.yml

```diff
@@ -536,6 +536,11 @@ enums:
         doc: |
           Enable passing SPIR-V shaders to @ref wgpuDeviceCreateShaderModule,
           via @ref WGPUShaderSourceSPIRV.
+      - name: reusable_adapters
```

Comment:
I just picked a name, but other suggestions welcome.

Reply:
Adapter expiry?

Reply:
Adapters could still "expire" in the sense that they got unplugged or something, which makes that naming kind of ambiguous. (Actually, the upstream spec uses both "consumed" and "expired" as separate things:
https://gpuweb.github.io/gpuweb/#dom-adapter-state-slot)
Regardless the most precise thing I could think of was `NoConsumeAdapter` or something but then we start getting double negatives ("NoConsumeAdapter is disabled") which is confusing.

Reply:
`MultiDeviceAdapter` maybe?

Reply:
That's a good direction. Updated with `MultipleDevicesPerAdapter`, I think that gets the point across pretty well.